src/link/batch/batching.ts

```diff
@@ -58,48 +65,70 @@ export class OperationBatcher {
   }
 
   public enqueueRequest(request: BatchableRequest): Observable<FetchResult> {
-    const requestCopy = {
+    const requestCopy: QueuedRequest = {
       ...request,
+      next: [],
+      error: [],
+      complete: [],
+      subscribers: new Set,
```

Comment:
I think you should call the constructor of the Set
```suggestion
      subscribers: new Set(),
```